### PR TITLE
Quantity.fromNumericalAmount, the inverse of getNumericalAmount

### PR DESCRIPTION
**Pull request by rhuffy**
## Description

We use getNumericalAmount to do some calculations on Quantity values as BigDecimals, and sometimes we want to convert back to a Quantity of the same units. This PR pulls the format multiplier math into a separate method and implements fromNumericalAmount as the inverse of getNumericalAmount.

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] Code contributed by me aligns with current project license: [Apache 2.0](https://www.apache.org/licenses/LICENSE-2.0)
 - [x] I Added [CHANGELOG](https://github.com/fabric8io/kubernetes-client/blob/master/CHANGELOG.md) entry regarding this change
 - [x] I have implemented unit tests to cover my changes
 - [x] I have added/updated the [javadocs](https://www.javadoc.io/doc/io.fabric8/kubernetes-client/latest/index.html) and other [documentation](https://github.com/fabric8io/kubernetes-client/blob/master/doc/CHEATSHEET.md) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=fabric8io_kubernetes-client) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: kubernetes-model-generator/kubernetes-model-core/src/test/java/io/fabric8/kubernetes/api/model/QuantityTest.java [hunk 1]
```diff
@@ -198,4 +198,40 @@ public void testIndexOfUnit() {
     assertEquals(4, Quantity.indexOfUnit("123 K"));
     assertEquals(4, Quantity.indexOfUnit("123c"));
   }
+
+  @Test
+  @DisplayName("Test fromAmountInBytes method")
+  public void testFromAmountInBytes() {
```
Comment (rohanKanojia) [inline kubernetes-model-generator/kubernetes-model-core/src/test/java/io/fabric8/kubernetes/api/model/QuantityTest.java:204]: Could you please make this test parameterized instead?

Comment (rhuffy) [reply kubernetes-model-generator/kubernetes-model-core/src/test/java/io/fabric8/kubernetes/api/model/QuantityTest.java]: thanks for the suggestion. I made the change.

**Review by shawkins — APPROVED** [review]
LGTM

**Comment by rohanKanojia** [comment]
@rhuffy : Could you please address this minor sonar smell?

**Review by rohanKanojia — APPROVED** [review]

**Comment by rhuffy** [comment]
Can we get another CI run on this? I believe I've fixed the test failures.

**Comment by rhuffy** [comment]
bump

**Comment by rhuffy** [comment]
bump @rohanKanojia

**Review by manusa — APPROVED** [review]

**Pull request merged**